src/main/scala/com/ebiznext/comet/job/index/bqload/BigQueryLoadJob.scala

```diff
@@ -171,12 +186,78 @@ class BigQueryLoadJob(
       logger.info(
         s"BigQuery Saved to ${table.getTableId} now contains ${stdTableDefinitionAfter.getNumRows} rows"
       )
+
+      prepareRLS().foreach { rlsStatement =>
+        logger.info(s"Applying security $rlsStatement")
+        try {
+          bqJobRun(rlsStatement)
+        } catch {
+          case e: Exception =>
+            e.printStackTrace()
+        }
+      }
       SparkJobResult(session)
     }
   }
 
+  private def bqJobRun(statement: String) = {
+    import java.util.UUID
+
+    import scala.collection.JavaConverters._
+    val jobId = JobId
+      .newBuilder()
+      .setJob(UUID.randomUUID.toString)
+      .setLocation(cliConfig.getLocation())
+      .build()
+    val config =
+      QueryJobConfiguration
+        .newBuilder(statement)
+        .setUseLegacySql(false)
+        .build()
+    // Use standard SQL syntax for queries.
+    // See: https://cloud.google.com/bigquery/sql-reference/
+    val job: Job = bigquery.create(JobInfo.newBuilder(config).setJobId(jobId).build)
+    scala.Option(job.waitFor()) match {
+      case None =>
+        throw new RuntimeException("Job no longer exists")
+      case Some(job) if job.getStatus.getExecutionErrors() != null =>
+        throw new RuntimeException(job.getStatus.getExecutionErrors().asScala.reverse.mkString(","))
+      case Some(_) =>
+      // everything went well !
+    }
+  }
+
+  private def toBQDelGrant(): String = {
```

Comment:
We're in BigQueryLoadJob, so we know that's a bigquery grant method, you can just name the method 
```suggestion
  revokeAllPrivileges
```

Reply:
Done !